### PR TITLE
added blue, red, and auto waypoints class to constants

### PR DESCRIPTION
**Pull request by bdan-codes**
I created the Red and Blue team constants, as well as a class that upon execution, grabs to correct values given the current team, making grabbing the waypoint constants easy peasy

### CODE REVIEW

**Comment by bdan-codes** [comment]
*only the constants file should be changed

**Comment by bdan-codes** [comment]
also idk if my solution is good, i am new to java so I probably did an illegal code thing or smth. anyways build failed so...

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -156,4 +164,55 @@ public static class AutoConstants {
...
+  }
+
+  public static class TeamWaypoints {
+    public static Pose2d speaker;
+    public static Pose2d amp;
+    public static Pose2d pickup;
+    public static Pose2d chainLeft;
+    public static Pose2d chainRight;
+    public static Pose2d chainBack;
+    public static void main(String[] args) {
```
Comment (josephhadad05) [inline src/main/java/frc/robot/Constants.java:187]: Don't make this a main method, maybe a constructor... If you do that, make sure to make the variables non-static.

Someone would get a variable like this.

```
Pose2d const = new TeamWaypoints().speaker;
```

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -156,4 +164,55 @@ public static class AutoConstants {
...
+    public static Pose2d chainBack = new Pose2d(6.40,4.10,new Rotation2d(90.00));
+  }
+
+  public static class RedTeamWaypoints {
+    public static Pose2d speaker = new Pose2d(15.10,5.65,new Rotation2d(90.00));
+    public static Pose2d amp = new Pose2d(14.85,7.15,new Rotation2d(180));
+    public static Pose2d pickup = new Pose2d(1.20,1.75,new Rotation2d(150.00));
+    public static Pose2d chainLeft = new Pose2d(12.55,2.65,new Rotation2d(-150.00));
+    public static Pose2d chainRight = new Pose2d(12.60,5.50,new Rotation2d(150.00));
+    public static Pose2d chainBack = new Pose2d(10.15,4.00,new Rotation2d(90.00));
```
Comment (josephhadad05) [inline src/main/java/frc/robot/Constants.java:176]: Rotation2d takes a value in radians

**Review by josephhadad05 — CHANGES_REQUESTED** [review]
For the most part pretty good. Few small changes needed. Ignore the build failed.

**Pull request merged**